monocle/worker.py

```diff
@@ -786,9 +786,10 @@ def travel_speed(self, point):
 
                 if (normalized not in SIGHTING_CACHE and
                         normalized not in MYSTERY_CACHE):
-                    if (encounter_conf == 'all'
-                            or (encounter_conf == 'some'
-                            and normalized['pokemon_id'] in conf.ENCOUNTER_IDS)):
```

Comment:
Why are the encounter configs removed? Would love to have them in addition to player level check. :(